recipes/lora_finetune_single_device.py

```diff
@@ -710,6 +721,8 @@ def recipe_main(cfg: DictConfig) -> None:
         - Overwritten by arguments from the command-line
     """
     config.log_config(recipe_name="LoRAFinetuneRecipeSingleDevice", cfg=cfg)
+    if cfg.get("low_cpu_ram", False):
+        common_utils._use_low_cpu_ram = True
```

Comment:
Does this have to be in `recipe_main`? Can we instead do it somewhere inside the recipe class (before the model gets instantiated)? Also would add a one-line comment explaining this